Floofbot/Services/EventLoggerService.cs

```diff
@@ -389,11 +389,18 @@ public Task UserLeft(IGuildUser user)
                         return;
 
                     var embed = new EmbedBuilder();
-
                     embed.WithTitle($"❌ User Left | {user.Username}#{user.Discriminator}")
                          .WithColor(Color.Red)
-                         .WithDescription($"{user.Mention} | ``{user.Id}``")
-                         .WithFooter($"user_leave user_joinlog {user.Id}")
+                         .WithDescription($"{user.Mention} | ``{user.Id}``");
+                    if (user.JoinedAt != null)
+                    {
+                        DateTimeOffset userJoined = ((DateTimeOffset)user.JoinedAt);
+                        TimeSpan interval = DateTime.UtcNow - userJoined.DateTime;
+                        string day_word = interval.Days == 1 ? "day" : "days";
+                        embed.AddField("Joined Server", userJoined.ToString("ddd, dd MMM yyyy"), true);
+                        embed.AddField("Time at Server", $"{interval.Days} {day_word}", true);
```

Comment:
Overall lgtm, however, wouldnt it be better to have it show years/months too? having, say, 760 days in the server is not very readable. Not PR blocking though just food for thought

Reply:
Hmm yes. You are right. Unfortunately `TimeSpan` object doesn't count years or months, so some arithmetic needs to be done here to implement that.